main.tf

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google-beta" {}
+
+resource "google_service_account" "bastion_host" {
+  project      = var.project
+  account_id   = "bastion"
+  display_name = "Service Account for Bastion"
+}
+
+resource "google_compute_instance" "bastion_vm" {
```

Comment:
Should we replace w/ https://github.com/terraform-google-modules/terraform-google-vm ?

Reply:
We should probably wait until https://github.com/terraform-google-modules/terraform-google-vm/pull/38 is merged before doing so as a bastion should probably have support for shielded vm.